Field.h

```diff
@@ -125,4 +126,38 @@ class TextField: public Field {
         string getValue() {
             return fieldValue;
         }
+};
+
+class CharField: public Field {
+    
+    private:
+        string fieldValue;
+        int maxLength;
+        
+    public:
+        CharField(string name, int maxLen, string defaultValue="", bool primary=false, bool unique=false, bool nullable=true):
+            Field(name, CHAR, primary, unique, nullable, maxLen) 
+        { 
+            fieldValue = defaultValue;
+            maxLength = maxLen;
+        } 
+        
+        void setValue(string value) {
+            setNull(false);
+            fieldValue = checkMaxLen(value, maxLength);
+        }
+        
+        string getValue() {
+            return fieldValue;
+        }
+        int getMaxLen() {
```

Comment:
Consistent spacing between methods. I have been using 1 newline between each method (I think - maybe just check the rest of the file to make sure), rather stick to that.